portfolio.html

```diff
@@ -23,88 +23,90 @@
 	<section id="topJump" class="mainImage">
 		<div class="backgroundImage">
 		<div class="overlayText">
-			<h1>Marcus Bowyer</h1>
+			<h1 class="fontStyleHead">Marcus Bowyer</h1>
 		<div class="secondOverlayText">
-			<h2>Full Stack Web Devloper</h2>
+			<h2 class="fontStyleHead">Full Stack Web Devloper</h2>
```

Comment:
Do you want your two headings here to have to be the same size? I'd consider making a separate class for this?

Reply:
They're not the same size as I overruled the size of h1 to 1.5rem in it's CSS further down. But I should probably change the class name as the class isn't just being used for headings anymore.